internal/pkg/manifest/validate_env.go

```diff
@@ -347,9 +350,13 @@ func (i RestrictiveIngress) validate() error {
 // validate returns nil if advancedCDNConfig is configured correctly.
 func (cfg advancedCDNConfig) validate() error {
 	if cfg.Certificate != nil {
```

Comment:
nit: can we flip the if-statement to create [a guard-clause](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html) to reduce nesting:

```go
if cfg.Certificate == nil { return nil }
// existing logic
```